### DISCUSSION TITLE
Upcoming Publication ...

### DISCUSSION THREAD

**Discussion by Alkibiades410** [Stellarium/stellarium#1284] [Q&A] [answered]
Howdy,

I have submitted a manuscript for publication, which makes _extensive_ use of Stellarium, and this includes several screenshots.  My attorney has reviewed the GNU General Public License v2.0 (GPL-2.0) under which the software still seems to fall (included with the program under GPL.txt).  Is this correct?  

I have also reviewed the topic under the included FAQ and the linked thread.  Nothings seems to have changed since that time.  

If true, then my use is covered under paras 0, 1, and 2.

I made sure none of the screenshots contained artwork (including textures), and I added the landscape (Athens, Greece) myself.  I expect this book will raise awareness of your outstanding work considerably among Classicists and Ancient Historians. 

My question is simple and straightforward:  I would like to add credit and appreciation under "Acknowledgements," how would the development team like this to read?

Thanks,
csp

ps -- you can view the pre-print blurb [here](https://www.academia.edu/37824514/The_Athenian_Year_Primer_Understanding_Attic_Time_Reckoning_and_the_Julian_Calendar).

**Comment by gzotti** [comment, marked as answer]
Hi!
A simple "Screenshot from Stellarium version 0.xx.y" would be nice in the image captions, and some citation of relevant papers by Stellarium team members. See User Guide bibliography, e.g. Zotti 2016b or 2019 (depending on what depth of simulation you used).  Or, our upcoming new paper just accepted for publication:

Georg Zotti, Susanne Hoffmann, Alexander Wolf, Fabien Chéreau, and Guillaume Chéreau. 
The simulated sky: Stellarium for cultural astronomy research. 
Journal for Skyscape Archaeology 6(2), in press.

I think we had contact some time ago. If we helped you by some explanation, or important operating details, I must leave words of acknowledgements to you.

**Comment by Alkibiades410** [comment]
Will do, and I am please to do so.  I have also incorporated the following blurb in Appx XI.  If you would, then please check to make sure I have said nothing stupidly inaccurate:
---------------------------------

**Stellarium**: At time of writing, many superb astronomy software programs existed, which incorporated everything explained in Part 2 and this appendix.  In fact, several possess capabilities and options far beyond what any archeoastronomer will need or probably use.

Moreover, one simply cannot go to Lykabettos Hill or the Pnyx today and watch the Athenian night sky:  all celestial objects have shifted, more or less, 30° since the timeperiod covered by the _Primer_.

I chose _Stellarium_ for a number of crucial reasons (in no particular order):  **i)** set location Lat/Long to the sec and **ii)** alt by meters above sea level; while using **iii)** Local True Solar Time; **iv)** render by stereographic projection (naked eye); **v)** add/remove light pollution and atmospheric refraction (by setting pressure, temperature, and extinction coefficient); **vi)** include nutation; **vii)** ready references to the Equation of Time and Julian Day conversions; **viii)** switching between VSOP87 / ELP2000-92B & DE 430 / 431; **ix)** adjust ΔT constant; **x)** catalogue over 210 million stars; and **xi)** render constellations, asterisms, and their boundaries.  

Without question, however, the most critical component:  project an accurate _Topographical Horizon_. When Ancient Greeks like Homer, Hesiod, Euktemon, Thoukydides, Theophrastos, or Ptolemy mention specific astronomical events, we must take into account locations like Khios, Boiotia, Attike, Lesbos, or Alexandria. 

A comprehensive and accurate _Visible Horizon_ provides the _sine qua non_ for timetravel back to Ancient Greece.  Without the collaborative effort of everyone involved, the _Primer_ would have never have left the drawing board.  I cannot express my gratitude strongly enough to all those, who developed the _Stellarium Project_ (see Bibliography Addendum p. 402).

**Comment by gzotti** [comment]
Sounds nice, thank you!
